Feature/GitHub discord integration

bodhisathwa opened the pull request:
This PR adds a Discord bot that integrates with GitHub to:
- Track and display contribution metrics
- Automatically manage Discord roles based on GitHub activity
- Provide real-time notifications of GitHub events
- Generate weekly activity summaries"

bodhisathwa commented:
Hello, RUXAILAB team!! I have done a working prototype of the GitHub - Discord Integration which shows the integration that automates role management, PR handling, and contributor analytics for Discord communities. I hope for review of this PR, Thank you.

marcgc21 reviewed (commented):
Hi @bodhisathwa! I started your review but I found so many binary files. Are you sure about that? Most of them came from __pycache__ folder so I assume that they shouldn't be there. 

Can you fix that?

marcgc21 reviewed (changes requested):
Let me know when you updated this 🙏

bodhisathwa commented:
Thank you @marcgc21 for the response!! I will make sure to update the files and commit them properly.

bodhisathwa commented:
Hello Mentor @marcgc21, I have updated the PR and deleted the binary files! I hope for the review of the PR. Thank you.

bodhisathwa commented:
> Let me know when you updated this 🙏

Hello Mentor @marcgc21, I have updated the PR and deleted the binary files! I hope for the review of the PR. Thank you.

marcgc21 commented:
This work was initially explored to address who best suits for the GSOC integration project, but now this topic will be handled by the selected GSoC 2026 student, we will defer to their implementation and direction.
